### PR TITLE
[newman-reporter-allure] Fix: Resolved the issue of formatting in JSON response content displaying in Allure reports.

### PR DESCRIPTION
**Pull request by win5923**
**Context**
When the `Content-Type` of a response body is `application/json`, the formatting should be normal.

**Screenshot**
Postman:
image
Postman Response body:
image

Allure report (anonymised for privacy):
image

#### Checklist
- [X] [Sign Allure CLA][cla]
- [ ] Provide unit tests

[cla]: https://cla-assistant.io/accept/allure-framework/allure2

### CODE REVIEW

**Review by epszaw — CHANGES_REQUESTED** [review]
Thank you for the contribution, good job here!
Please, provide unit-tests for the changes to continue with the PR.

**Comment by win5923** [comment]
I have completed the changed, and sorry for not being able to provide unit tests. I am unsure about how to proceed with them. 
Thank you very much for your review.

File: packages/newman-reporter-allure/src/index.ts [hunk 1]
```diff
@@ -369,13 +369,21 @@ class AllureReporter {
     }
 
     if (response?.body) {
+      let contentType = ContentType.TEXT;
+
+      // Check if the response content type is JSON
+      const contentTypeHeader = args.item.responses.members[0]?.getHeader('Content-Type');
+      if (contentTypeHeader && contentTypeHeader.toLowerCase().includes('application/json')) {
```
[the remark this answers is not in the document]
Comment (win5923) [reply packages/newman-reporter-allure/src/index.ts]: done!

**Comment by win5923** [comment]
@epszaw Is there anything else that needs to be modified?

**Comment by epszaw** [comment]
@win5923 please, check typescript errors and linters

**Pull request closed without merging**